Tutorials/whatiswot/3-JSON/3-JSON.txt

```diff
@@ -0,0 +1,46 @@
+In this video, we will take a closer look at JSON. Since it is the most used payload format in the Web of Things , as well as the way Thing Descriptions are represented. 
+
+So, let’s start with what is JSON. 
+
+JSON is a lightweight data-interchange format. 
+
+It is easy for humans to read and write 
+
+and easy for machines to parse and generate. 
+
+Let’s take a look at how parsing and generating happens. We can parse a JSON to visualize the data such as temperature or we can send data, such as form content as JSON. 
+
+Let’s look at the quick facts about the JSON.  
+
+It was standardized by ECMA and I E T F in around 2013, the current version is 2017 [RFC 8259]
+
+It is used for data exchange but also for configuration files and It can be translated to other popular formats in both ways like XML,  YAML, and other formats.
```

Comment:
The audio is different. The one in the text here is nicer (both ways before the format names)